patches/server/0464-Add-additional-open-container-api-to-HumanEntity.patch

```diff
@@ -3,47 +3,148 @@ From: JRoy <joshroy126@gmail.com>
 Date: Wed, 26 Aug 2020 02:12:31 -0400
 Subject: [PATCH] Add additional open container api to HumanEntity
 
+Expand existing methods to also allow for title overrides
 
+Co-authored-by: Lukas Planz <lukas.planz@web.de>
+
+diff --git a/src/main/java/net/minecraft/server/level/ServerPlayer.java b/src/main/java/net/minecraft/server/level/ServerPlayer.java
+index e087b62f88157d57eb037feb0d39c076b286f755..8325d03b8f1f08c7ef68aaf10f2e5b8d7d35675b 100644
+--- a/src/main/java/net/minecraft/server/level/ServerPlayer.java
++++ b/src/main/java/net/minecraft/server/level/ServerPlayer.java
+@@ -1502,7 +1502,7 @@ public class ServerPlayer extends Player {
+     }
+ 
+     @Override
+-    public OptionalInt openMenu(@Nullable MenuProvider factory) {
++    public OptionalInt openMenu(@Nullable MenuProvider factory, @Nullable net.kyori.adventure.text.Component titleOverride) { // Paper - overridable title
```

Comment:
I don't like calling this `titleOverride`. Even tho its just a param name, it should be something like `defaultTitleOverride` or something to distinguish it from the actual title override just added to InventoryOpenEvent